deployments/ocm-install/helmfile.yaml

```diff
@@ -145,71 +149,35 @@ releases:
                       "name": "oCIS Test",
                       "full_name": "oCIS Test provider",
                       "organization": "oCIS",
-                      "domain": "ocis.kube.owncloud.test",
-                      "homepage": "https://ocis.kube.owncloud.test",
+                      "domain": "*.kube.owncloud.test",
```

Comment:
needs to be something like:

`"domain": ".*\\.kube\\.owncloud\\.test"`

Double escaping is needed for:
1) JSON
2) regex